libs/sdk-ui-dashboard/src/presentation/topBar/buttonBar/button/cancelButton/types.ts

```diff
@@ -0,0 +1,16 @@
+// (C) 2022 GoodData Corporation
+
+import { ComponentType } from "react";
+
+/**
+ * @internal
+ */
+export interface ICancelButtonProps {
+    isVisible: boolean;
```

Comment:
Should this be part of the Button(s) API, or ButtonBar API only (which makes more sense in my point of view)?
I think that `isVisible` property makes sense for dialogs only, but I'm not sure...

Reply:
Makes sense, this is more convenient.